README.md

```diff
@@ -1,2 +1,41 @@
-# conduit-connector-amazon-sqs
-Conduit connector for Amazon SQS 
+# Conduit Connector for Amazon SQS 
+[Conduit](https://conduit.io) for [Amazon SQS](https://documentation.spire.com/maritime-2-0/).
+
+## How to build?
+Run `make build` to build the connector.
+
+
+## Source
+The source connector pulls data from the Amazon SQS Queue. As messages come in, the source connector grabs a single message from the Amazon SQS Queue, formats it for the destination connector as a new record, and sends it. The Message Body of the SQS Message is formatted into a record's payload, and the Message Attributes are passed as record metadata. After the destination connector acknowledges that it has received the record, the source connector deletes the message from the Amazon SQS Queue.
```

Comment:
> After the destination connector acknowledges

Nitpick:  there can be multiple connectors. I believe it's enough to say: "After the record has been acknowledged" without going into details.